polaris-pipeline/Common/Dto/Case/DefendantDetailsDto.cs

```diff
@@ -22,6 +23,32 @@ public class DefendantDetailsDto
         [JsonProperty("dob")]
         public string Dob { get; set; }
 
+        public string Age
+        {
+            get
+            {
+                // YYYY-MM-DD ?
+                if((Dob.Length == 10) 
```

Comment:
I'll leave this up to you, but you may want to use a Regex expression to ensure it is a valid date before passing the values on to the calculation?